test/views/TwoEyeReviewSpec.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import play.api.inject.Injector
+import play.api.i18n.{Messages, MessagesApi, Lang}
+import play.api.test.FakeRequest
+import play.twirl.api.Html
+import java.time._
+import org.jsoup._
+import views.html._
+import org.jsoup.nodes._
+import scala.collection.JavaConverters._
+import config.AppConfig
+import models._
+import PageReviewStatus._
+
+class TwoEyeReviewSpec extends ViewSpecBase {
+
+  def elementAttrs(el: Element): Map[String, String] = el.attributes.asScala.toList.map(attr => (attr.getKey, attr.getValue)).toMap
+
+  trait Test {
+    private def injector: Injector = app.injector
```

Comment:
Some of the attributes declared here are already defined in ViewSpecBase. I'll leave it up to you as to whether, or not, you want to override the values.

Reply:
removed local versions